dev-packages/e2e-tests/test-applications/create-remix-app/playwright.config.ts

```diff
@@ -57,7 +57,7 @@ const config: PlaywrightTestConfig = {
       port: eventProxyPort,
     },
     {
-      command: `PORT=${port} pnpm start`,
+      command: `PORT=${port} NODE_OPTIONS='--require ./instrument.server.cjs' pnpm start`,
```

Comment:
Is it necessary to run it like this, does it not work with require on top? Just to clarify!

Reply:
Yes, we need to run it like this. esbuild does not bundle the instrumentation inside the entry files correctly.

Reply:
hmmm I wonder if this qualifies as a breaking change 😬 thoughts cc @AbhiPrasad ?

Reply:
I think this does qualify as a breaking change 😬

I guess we have to opt-in to this mode, we can't break the people who already put the work in to upgrade their sdk to v8.

Reply:
We can make this the default during onboarding though.

Reply:
Can we try adding another `init` option like `useOtel` to switch to this mode, and initialise the integrations without (or also with) exposing them? 

This change also doesn't affect Express servers (or I guess any other custom server), a top-level `import` on the server file works in those cases. So we can still get rid of the Express adapter instrumentation, and keep the core instrumentation as an alternative for `built-in` server users (which I also _**guess**_ is not very popular among Remix users looking at our issue reports)

I have updated the e2e tests using Express custom servers. (https://github.com/getsentry/sentry-javascript/pull/12110/commits/0e2f73aabaadf4aa64c952e7aaf85353cf423699)